egs/iam/s5/local/unk_arc_post_to_transcription.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+import argparse
```

Comment:
will do

Reply:
@danpovey BTW after I figured out the purpose and context of this script, I agree that its function definitely makes sense (I've asked Ashish to add more explanation though). There's no other existing solution for now. I have some scripts (not committed yet) which pick up unks from arc-post files, and then insert recognized unks from P2G into a ctm. But they are not very related since we don't need P2G here.